yellowbrick/classifier/class_balance_heat_map.py

```diff
@@ -0,0 +1,169 @@
+# yellowbrick.classifier.class_balance_heat_map
+# Class balance visualizer for showing per-class support.
+#
+# Author:   Rebecca Bilbro <rbilbro@districtdatalabs.com>
+# Author:   Benjamin Bengfort <bbengfort@districtdatalabs.com>
+# Author:   Neal Humphrey
+# Author:   Larry Gray
+# Created:  Sun March 4 20:45:00 2018 -0400
+#
+# Copyright (C) 2017 District Data Labs
+# For license information, see LICENSE.txt
+#
+# ID: class_balance_heat_map.py [5388065] lwgray@gmail.com $
+
+"""
+Class balance heat map visualizer for showing per-class support.
+"""
+
+##########################################################################
+## Imports
+##########################################################################
+
+from matplotlib import cm
+import matplotlib.pyplot as plt
+import numpy as np
+
+from .base import ClassificationScoreVisualizer
+
+from sklearn.model_selection import train_test_split
+from sklearn.utils.multiclass import unique_labels
+
+
+##########################################################################
+## Class Balance Heat Map Chart
+##########################################################################
+
+class ClassBalanceHeatMap(ClassificationScoreVisualizer):
```

Comment:
If we do decide to make this a separate class, I recommend that we make it a subclass of `ClassBalance` 

Also, I'd like to talk about the name - this is technically a heat map since it is using different colors to represent data, however, I'm afraid that it implies that the strength or hue of the color implies the value's magnitude. Because we're talking about classes, they are discrete. 

For me, programming design is about the naming of things; once you nail the name function follows. If we can't think of a good name for this, then I believe that strongly implies we make this an option of `ClassBalance` rather than an independent visualizer. 

My suggested names are:

- `PredictionDistribution` 
- `PredictionBalance` 
- `ClassPredictionError` 

Thoughts? Do any of these things jump out at you?

Reply:
I'm split between ClassPredictionError and PredictionDistribution- but I think the former is a more informative name.